ngraph_bridge/ngraph_backend.h

```diff
@@ -0,0 +1,185 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
```

Comment:
Add Header guards for the new files?

Reply:
Are header guards needed? We've `#pragma once`.

Reply:
Not sure which is better to use, we have had both in the files.

Reply:
We can get rid of header guards in a separate PR. Almost all modern compilers support `#pragma once`.